KIF Tests/PickerTests_ViewTestActor.m

```diff
@@ -107,7 +107,7 @@ - (void)testSelectingRowInComponent
     [[viewTester dateSelector] tap];
     NSArray *date = @[ @"December", @"31", @"2030" ];
     [viewTester selectDatePickerValue:date];
-    [viewTester selectPickerViewRowWithTitle:@"17" inComponent:1];
+    [viewTester selectDatePickerViewRowWithTitle:@"17" inComponent:1];
     [[[viewTester dateSelector] usingValue:@"Dec 17, 2030"] waitForView];
```

Comment:
@justinseanmartin Made those changes you suggested. This test was failing because with the new changes inside `KIFUIViewTestActor`, the var `viewTester` would expect a picker view and not a date picker based on the expected class predicate. We decided the least complex and most clear change would be to add this function `selectDatePickerWithRowWithTitle:inComponent` to change individual components in a date picker. Changing individual components of regular pickers would use the original  `selectPickerViewRowWithTitle :inComponent`. Does this look okay to you?